### PR TITLE
Enable Explore page for MNW

### PR DESCRIPTION
**Pull request by followdarko**
Enable Explore page for MNW after solving img rendering issue

Resolved img rendering issue could see here: https://preflight.mynearwallet.com/explore

Screenshot 2022-09-07 at 13 11 04

### CODE REVIEW

**Review by roshaans — APPROVED** [review]
LGTM 🚀 Nice job :)

**Review by roshaans — APPROVED** [review]
LGTM 🚀

File: packages/frontend/src/components/explore/ExploreContainer.js [hunk 1]
```diff
@@ -30,13 +27,10 @@ const StyledH1 = styled.h1`
 `;
 
 export function ExploreContainer() {    
-    const nearBalance = useSelector(selectAvailableBalance) || 0;
```
[the remark this answers is not in the document]
Comment (PinkiNice) [reply packages/frontend/src/components/explore/ExploreContainer.js:33]: Due to loading delay behaviour of a banner was unpleasant (bumpy).
So for now we are showing it constantly, yeah
Until we have time to reimplement balance loading or come up with better ideas here

**Review by Patrick1904 — APPROVED** [review]
LGTM

**Pull request merged**